letsencrypt-auto

```diff
@@ -3,16 +3,29 @@
 # A script to run the latest release version of the Let's Encrypt in a
 # virtual environment
 #
+# ./letsencrypt-auto     --      run the latest public release from PyPI
+#
+# ./lesencrypt-dev       --      run the copy from the local tree, including
```

Comment:
I strongly feel that these should be separate scripts. As I've noted before, I believe a non-negligible fraction of our beta users will be reading this script to see what the hell we are doing to their servers when we are automatically downloading, installing, upgrading, and running the client. This script should be as small, concise, and readable as possible and making this script double as a script for developers moves in the opposite direction.